service-catalog/components/peribolos-as-a-service.yaml

```
@@ -7,6 +7,7 @@ metadata:
     operate-first.cloud/logo-url: https://raw.githubusercontent.com/operate-first/peribolos-as-a-service/main/static/robot.png
     argocd/app-name: peribolos-as-a-service-smaug
     github.com/project-slug: operate-first/peribolos-as-a-service
+    backstage.io/kubernetes-label-selector: 'app.kubernetes.io/name: peribolos-as-service'
```

This is not a valid label selector. I think it should be:

```suggestion
    backstage.io/kubernetes-label-selector: 'app.kubernetes.io/name=peribolos-as-service'
```

---

Yea thats right, I just copied it without checking. I will pay more attention next time.